datasets/generics_kb/README.md

```diff
@@ -0,0 +1,186 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- en
+licenses:
+- cc-by-4.0
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- original
+task_categories:
```

Comment:
From the paper - "_it can assist in both question-answering and explanation construction for two existing datasets__"  Adding question-answering tags to task categories & task ids, will help in discovery on these datasets. 
Refer the dataset viewer app for the tast-categories : https://huggingface.co/datasets/viewer/

Reply:
I considered adding Question-answering, but am not sure of it can directly be applied, because the datasets do not include questions/answer pairs. Its more about eugenic truth sentences. Thus, had added 'others' as tags so the experts could review.